### PR TITLE
#163: Show latest metric only

### PR DESCRIPTION
**Pull request by pippinlee**
JIRA Ticket link:

https://github.com/dessa-oss/atlas/issues/163

### CODE REVIEW

**Comment by choi-calvin** [comment]
Sorry I'm a bit out of the loop on this. Was this the recommended UX solution from the MLE's?

**Comment by pippinlee** [comment]
Hopefully this summarizes well: when logging a metric continuously throughout the training, MLE's primarily look for the last metric when inspecting a job run. So even though continuous logging of a metric would create a list, we can ignore all metrics except the last as all other metrics are process details and would make for a tougher time reading results.

**Comment by choi-calvin** [comment]
Sorry let me rephrase, the conversation in the ticket seems inconclusive. Were there any conclusions made?

**Comment by mohammedri** [comment]
@choi-calvin the conclusion is to change the functionality of log_metrics such that only the most recent metric published to it is shown in the UI. 

This is the task to be able to do so

**Comment by choi-calvin** [comment]
Got it, thanks for the clarification

**Review by choi-calvin — APPROVED** [review]

**Comment by mohammedri** [comment]
@pippinlee OK to merge this?

**Comment by pippinlee** [comment]
Let's hold off on merging till UAT pipeline is green: https://jenkins.shehanigans.net/job/foundations-uat/
